controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -223,6 +224,10 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			return ctrl.Result{}, err
 		}
 
+		if err := r.configureMonitoring(ctx, instance); err != nil {
```

Comment:
can we rename this function to something e.g `monitoringCR()`

Reply:
also this should be only called either
`case cluster.SelfManagedRhoai`
or 
`case cluster.ManagedRhoai:`
right?
so ODH case, we no need create such CR even DSCI has monitoring to Managed